web/web.go

```diff
@@ -263,7 +265,20 @@ func (h *Handler) statush(w http.ResponseWriter, r *http.Request) {
 	h.status.mu.RLock()
```

Comment:
Can't comment on the right line, but btw., is the method above really meant to be called `statush` vs. `status`?